.github/workflows/spellcheck.yml

```
@@ -0,0 +1,24 @@
+name: Spellcheck
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

v3 exists and as this is being introduced for the first time would be nice to start modern

```suggestion
        uses: actions/checkout@v3
```

---

Ha, that must have happened since I created this!